hazelcast/src/test/java/com/hazelcast/internal/ascii/MemcachedTest.java

```diff
@@ -57,20 +57,23 @@
 @Category(QuickTest.class)
 public class MemcachedTest extends HazelcastTestSupport {
 
-    protected Config config = new Config();
     protected HazelcastInstance instance;
     protected MemcachedClient client;
 
-    @Before
-    public void setup() throws Exception {
+    protected Config createConfig() {
+        Config config = new Config();
```

Comment:
As an improvement over current state of this test, could initialize config with `smallInstanceConfig()` (also use the same in `MemcachedMultiendpointTest`)